.github/workflows/e2e_libvirt.yaml

```diff
@@ -117,6 +121,8 @@ jobs:
           # For debugging
           ls ./target/release
           popd
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Really interesting that `gh` doesn't read from `GITHUB_TOKEN`. Never used that tool much so don't know for sure.